File: java2csharp/java/Samsung/RichNotifications/MyCallbackReceiver.java

package com.samsung.android.richnotification.sample;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;
import android.widget.Toast;

public class MyCallbackReceiver extends BroadcastReceiver {

    @Override
    public void onReceive(Context context, Intent intent) {

        String data = intent.getStringExtra("extra_action_data");

        if (data != null) {
            Toast.makeText(context, data, Toast.LENGTH_SHORT).show();
        }
    }
}

File: java2csharp/csharp/Samsung/RichNotifications/MyCallbackReceiver.cs
namespace com.samsung.android.richnotification.sample
{

	using BroadcastReceiver = android.content.BroadcastReceiver;
	using Context = android.content.Context;
	using Intent = android.content.Intent;
	using Toast = android.widget.Toast;

	public class MyCallbackReceiver : BroadcastReceiver
	{

		public override void onReceive(Context context, Intent intent)
		{

			string data = intent.getStringExtra("extra_action_data");

			if (data != null)
			{
				Toast.makeText(context, data, Toast.LENGTH_SHORT).show();
			}
		}
	}

}